### DISCUSSION TITLE
Where is Deferral Operator / Trigger log in Standalone mode

### DISCUSSION THREAD

**Discussion by zhuowang-linkedin** [apache/airflow#23310] [Q&A]
seems no where to find the trigger log file when a task is deferred, the task log only shows error Unknown. not debug friendly for trigger dev.

**Comment by potiuk** [comment]
Airflow in standalone mode print logs in the stdout. Standalone mode is really for running it at command line for quick testing. Log fles are really only useful when you run airflow as separate components in "production" mode.
